web/modules/custom/dpl_consumers/dpl_consumers.deploy.php

```diff
@@ -22,8 +23,28 @@
  * as we don't want to create duplicates.
  */
 function dpl_consumers_deploy_10001(): void {
-  dpl_consumers_delete_user();
-  dpl_consumers_delete_consumer();
-  dpl_consumers_create_user();
-  dpl_consumers_create_consumer();
+  // Noop. We don't need to do anything here.
+  // Since we have changed our minds.
+  // @see dpl_consumers_deploy_10002().
+}
+
+/**
+ * Expand the users, roles, consumers so they can handle both BNF and GO.
+ *
+ * So we delete previous entities and create new ones.
+ */
+function dpl_consumers_deploy_10002(): void {
+  // Delete consume and users that we want to handle differently.
+  // We want to create consumers and consumer users
+  // specifically for the two known consumers (BNF and Go) and connected users.
+  (new Consumer("graphql_consumer", \Drupal::entityTypeManager()))->delete();
```

Comment:
Maybe `ConsumerHandler` should have methods for getting `Consumer` and `ConsumerUser` so the DI can be passed to it? Having to create a repository service for each of them is a bit overkill, but I think having to pass `entityTypeManager` to each `new` is a bit of a DX wart.